hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1249,10 +1256,14 @@ public String listStatus(final Path path, final String startFrom,
             startFrom);
 
     final String relativePath = getRelativePath(path);
+    AbfsClient listingClient = getClient();
 
     if (continuation == null || continuation.isEmpty()) {
       // generate continuation token if a valid startFrom is provided.
       if (startFrom != null && !startFrom.isEmpty()) {
+        // Blob Endpoint Does not support startFrom yet.
+        // Fallback to DFS Client for listing with startFrom.
```

Comment:
Also mention that this is internal use only, HDFS APIs cant pass startFrom marker today.

Reply:
Added in comments